ecosystem/sep_identicons.md

```diff
@@ -0,0 +1,97 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Identicons for Stellar Accounts
+Author: Lobstr.co
```

Comment:
It would be helpful if this field could mention individuals along with GitHub handles, however if you'd prefer to keep it with a company name that's fine.
```suggestion
Author: Lobstr.co, Gleb Pitsevich (@pitsevich)
```